src/oemof/tabular/facades.py

```diff
@@ -699,49 +699,34 @@ class ExtractionTurbine(ExtractionTurbineCHP, Facade):
     ...     thermal_efficiency=0.35)
 
     """
+    carrier: str
 
-    def __init__(self, *args, **kwargs):
-        kwargs.update(
-            {
-                "_facade_requires_": [
-                    "fuel_bus",
-                    "carrier",
-                    "tech",
-                    "electricity_bus",
-                    "heat_bus",
-                    "thermal_efficiency",
-                    "electric_efficiency",
-                    "condensing_efficiency",
-                ]
-            }
-        )
-        super().__init__(
-            conversion_factor_full_condensation={}, *args, **kwargs
```

Comment:
Need to incorporate this. Suggestion: Introduce conversion_factor_full_condensation as an attribute with default factory dict.